MediaManager/Plugin.MediaManager.iOS/AudioPlayerImplementation.cs

```diff
@@ -218,7 +218,10 @@ private void InitializePlayer()
             if(mediaFile != null)
                 nsUrl = new NSUrl(mediaFile.Url);
 
-            if (Status == MediaPlayerStatus.Paused)
+            var currentPlayerAsset = (AVUrlAsset) Player.CurrentItem?.Asset;
+            var currentPlayerItemUrl = currentPlayerAsset?.Url?.ToString();
+
+            if (Status == MediaPlayerStatus.Paused && mediaFile.Url == currentPlayerItemUrl)
```

Comment:
I think the bigger question here is what identifies a MediaFile uniquely and how we should compare them.